Formula/ooniprobe.rb

```diff
@@ -190,35 +223,35 @@ def plist; <<-EOS.undent
    <dict>
      <key>Label</key>
        <string>#{plist_name}</string>
-     <key>Program</key>
-       <string>#{opt_bin}/ooniprobe</string>
      <key>ProgramArguments</key>
      <array>
-       <string>-i</string>
-       <string>#{opt_share}/ooni/decks/current.deck</string>
+       <string>#{opt_bin}/ooniprobe-agent</string>
+       <string>start</string>
      </array>
      <key>RunAtLoad</key>
-       <false/>
+       <true/>
      <key>KeepAlive</key>
-       <false/>
+       <true/>
+     <key>WorkingDirectory</key>
+       <string>#{var}/log/</string>
      <key>StandardErrorPath</key>
-       <string>/dev/null</string>
+       <string>#{var}/log/ooniprobe.log</string>
      <key>StandardOutPath</key>
-       <string>/dev/null</string>
-     <key>StartCalendarInterval</key>
-     <dict>
-       <key>Hour</key>
-       <integer>00</integer>
-       <key>Minute</key>
-       <integer>00</integer>
-     </dict>
+       <string>#{var}/log/ooniprobe.log</string>
    </dict>
    </plist>
    EOS
   end
 
   test do
+    # Initialize ooniprobe to skip informed consent when testing
+    puts "#{var}/run/ooni/initialized"
+    is_initialized = File.exist?("#{var}/run/ooni/initialized")
+    (var/"run/ooni/initialized").write "" unless is_initialized
```

Comment:
OK I will think of some way to overcome this and generate a configuration file specific for testing.